InvenTree/part/helpers.py

```diff
@@ -0,0 +1,64 @@
+"""Various helper functions for the part app"""
+
+import logging
+
+from jinja2 import Environment
```

Comment:
Question: Is there a performance reason you are using Jinja2 instead of Djangos Template engine (which is based on Jinja2)? This will mean the syntax between reports and this field is different. I have not seen significant timing differences between Jinja2 and djangos template language when cached.

Reply:
@matmair this is specifically not to break current functionality. We have been using `jinja2` here for a long time - although I cannot remember *why*. 

However changing the rendering engine at this stage would break the current default template string - `jinja2` supports some tag attributes which the Django engine does not

Reply:
I'd be happy to look at switching to Django template engine in a separate PR